### PR TITLE
[BUGFIX] Fix interlink generation

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: packages/typo3-docs-theme/resources/config/typo3-docs-theme.php [hunk 1]
```diff
@@ -41,6 +44,10 @@
         ->set(IssueReferenceTextRole::class)
         ->tag('phpdoc.guides.parser.rst.text_role')
         ->set(phpDocumentor\Guides\ReferenceResolvers\Interlink\DefaultInventoryLoader::class)
+        ->set(Typo3InventoryRepository::class)
+        ->arg('$inventoryConfigs', param('phpdoc.guides.inventories'))
+        ->set(InventoryRepository::class, Typo3InventoryRepository::class)
+        ->arg('$inventoryConfigs', param('phpdoc.guides.inventories'))
```
Comment (garvinhicking) [inline packages/typo3-docs-theme/resources/config/typo3-docs-theme.php:50]: Hm, shouldn't line 47+48 be superfluous, and 49+50 alone suffice?

File: Documentation/Index.rst [hunk 2]
```diff
@@ -45,32 +45,3 @@ to extend the tool.
     Developer/Index
     Migration/Index
     KnownProblems/Index
-
```
Comment (garvinhicking) [inline Documentation/Index.rst:48]: I think this block may need a rebase?

**Review by garvinhicking — COMMENTED** [review]
Some questions

File: packages/typo3-docs-theme/resources/config/typo3-docs-theme.php (hunk 1 above)
Comment (linawolf) [reply packages/typo3-docs-theme/resources/config/typo3-docs-theme.php:50]: Ack

File: Documentation/Index.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/Index.rst:48]: What do you mean?

**Comment by linawolf** [comment]
@garvinhicking can we merge this to get rid of the warnings?

**Review by brotkrueml — APPROVED** [review]

File: Documentation/Index.rst (hunk 2 above)
Comment (garvinhicking) [reply Documentation/Index.rst]: It got rebased now, all good now it seems.

**Review by garvinhicking — APPROVED** [review]
When merged, maybe you can rebase the PR for the "test-docs" one too and merge it (or approve and I merge), thanks!

**Pull request merged**